lib/ClangImporter/ClangImporter.cpp

```diff
@@ -984,6 +984,21 @@ ClangImporter::getClangArguments(ASTContext &ctx) {
   return invocationArgStrs;
 }
 
+/// Gather all -Xclang flags from a Clang driver command-line
+static void extractClangFrontendInvocationArgs(const ArrayRef<std::string> invocationArgStrs,
```

Comment:
I think `ArrayRef` is an immutable data structure. Is there a particular reason to make it `const` explicitly?

Reply:
None. Removed.